plugins/modules/pulp_file_remote.py

```diff
@@ -120,7 +130,7 @@ def main():
         'name': module.params['name'],
     }
     desired_attributes = {
```

Comment:
Actually, I am quite pleased to see, that this is all it needs at the implementation side. ;)